sktime/forecasting/ttm.py

```diff
@@ -35,6 +35,21 @@ class TinyTimeMixerForecaster(_BaseGlobalForecaster):
     open-sourced by IBM Research. With less than 1 Million parameters, TTM introduces
     the notion of the first-ever "tiny" pre-trained models for Time-Series Forecasting.
 
+    **Zero-shot and Fine-tuning**:
+    The Model serves as a foundation model capable of both zero-shot forecasting and
+    fine-tuning on custom datasets. The decision to fine-tune is handled internally and
+    is determined by the configuration specified during model loading.
+    - **Zero-Shot Forecasting**: When the default configuration remains unchanged,
```

Comment:
you need to leave space between bullet points, or it will not render in rst - have a look in the detailed docs.

Reply:
added